components/_patterns/02-molecules/date-bars/date-bar.twig

```diff
@@ -0,0 +1,47 @@
+{#
+/**
+ * @file
+ * Event address bar for displaying date and time information for events.
+ * 
+ * Variables:
+ * # event_date_start — The start date/time.
+ * # event_date_end — The end date/time.
+ * # event_time_range — String with the time range.
+ */
+#}
+
+{% set date_bar_base_class = 'date-bar' %}
+
+{% if date(event_date_end) < date() %}
+  {% set date_bar_modifiers = ['ended'] %}
+{% endif %}
+
+<div {{ bem(date_bar_base_class , (date_bar_modifiers), date_bar_blockname) }}>
+  {% if date(event_date_end) < date() %}
+    The event has ended
+  {% elseif event_date_end|date('Y-m-d') > event_date_start|date('Y-m-d') %}
+    Open now through
+    {% include "@atoms/02-text/00-headings/_heading.twig" with {
```

Comment:
Now that I see this in action, I'm curious why it's an `h2` instead of just text? It doesn't look to me that it's semantically a heading. Can you explain why you all decided on `h2` instead of just a `div`? I assume I just don't have the bigger picture in mind, and am missing something.

To me, it looks semantically like meta-data about the event. The heading would be the title of the event (which now that I'm looking at it, I don't even see in the designs... Maybe it's the name of the location? That _looks_ more like a heading to me... Oh, or maybe the event title is somewhere else on the page since this is going to show in the side bar - I think?)

I think I'd like to talk this through tomorrow, instead of just typing my stream of consciousness...